pkgs/development/compilers/koka/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, gcc, cmake, makeWrapper, mkDerivation, fetchFromGitHub
+, alex, array, base, bytestring, cond, containers, directory, extra
+, filepath, haskeline, hpack, hspec, hspec-core, json, lib, mtl
+, parsec, process, regex-compat, text, time }:
+
+let
+  version = "2.1.1";
+  src = fetchFromGitHub {
+    owner = "koka-lang";
+    repo = "koka";
+    rev = "v${version}";
+    sha256 = "sha256-cq+dljfTKJh5NgwQfxQQP9jRcg2PQxxBVEgQ59ll36o=";
+    fetchSubmodules = true;
+  };
+  kklib = stdenv.mkDerivation {
+    pname = "kklib";
+    inherit version;
+    src = "${src}/kklib";
+    nativeBuildInputs = [ cmake ];
+  };
+  runtimeDeps = [ cmake gcc ];
```

Comment:
Can you check if `stdenv.cc` works here instead if you do `koka.override { stdenv = clangStdenv; }` would be pretty cool if it could support clang.

Reply:
Yeah, works here since on macOS it uses clangStdenv.